ILSpy/Themes/SyntaxColor.cs

```diff
@@ -0,0 +1,32 @@
+#nullable enable
+
+using System.Windows;
+using System.Windows.Media;
+
+using ICSharpCode.AvalonEdit.Highlighting;
+
+namespace ICSharpCode.ILSpy.Themes;
+
+public class SyntaxColor
+{
+	public Color? Foreground { get; set; }
+	public Color? Background { get; set; }
+	public FontWeight? FontWeight { get; set; }
+	public FontStyle? FontStyle { get; set; }
+
+	public void ApplyTo(HighlightingColor color)
+	{
+		color.Foreground = Foreground is { } foreground ? new SimpleHighlightingBrush(foreground) : null;
```

Comment:
just a small nit: why use pattern matching here? The lhs of the pattern is already a correctly typed variable. Wouldn't `Foreground is not null ? new SimpleHighlightingBrush(Foreground) : null` be easier to read?

Reply:
No good reason, I think I just started to use pattern matching a bit more than necessary recently. 😅